test/chpldoc/errors/saveSphinxInDocs.doc.chpl

```diff
@@ -0,0 +1,6 @@
+/* This is a routine called foo() */
+proc foo() {
```

Comment:
Ideally, I'd sort this into the same directory as the other --save-sphinx tests (though I could see that causing interference depending on when the test is run relative to other tests in that directory, since it modifies the docs/ directory.  So maybe give that directory its own error subdirectory for it?)

Reply:
How important to you is this?  Currently, it looks like there are ~25 directories that test `--save-sphinx` (and personally, it'd never have occurred to me to look for save-sphinx tests in a subdirectory named `folder/`).

Reply:
> Currently, it looks like there are ~25 directories that test `--save-sphinx`

There's a difference between using a flag because it makes comparing output easier and using a flag because the tests are specifically for that flag.  I didn't particularly want to maintain .good files that were filled with html cruft that was likely to change with Sphinx versions, and we moved away from .txt outputs by default, which left the output from --save-sphinx as the best option.  I don't think that's a reason not to put this test in the suggested directory, since its purpose is to exercise the flag itself.

> (and personally, it'd never have occurred to me to look for save-sphinx tests in a subdirectory named `folder/`).

I mean, I can go into why I chose that name for that overarching directory.  Or you can take the subdirectories out of it and make them all directly under the compflags folder if you think that would be better I don't particularly care that much.  I just think similar tests should be sorted together, and I think "errors" is too broad a category.

Reply:
That's why I asked how important it was to you:  To me, this is a cross-cutting concern ("it's an error";  "but it's also a test of how `--save-sphinx` works"), so there's no single correct place to put it.  I wouldn't have found your preferred place with the current directory naming structure (I did look for it) so was curious whether you were insistent.  I've now moved it since it seems like you were.